site/src/modules/navigation.ts

```diff
@@ -27,8 +27,13 @@ export const linkToUsers = withFilter("/users", "status:active");
 
 export const linkToTemplate =
   (organizationName: string, templateName: string): LinkThunk =>
-  (dashboard) =>
-    dashboard.experiments.includes("multi-organization") &&
-    selectFeatureVisibility(dashboard.entitlements).multiple_organizations
+  (dashboard) => {
+    const hasMultipleOrganizations = dashboard.organizations.length > 1;
+    const organizationsEnabled =
+      dashboard.experiments.includes("multi-organization") &&
+      selectFeatureVisibility(dashboard.entitlements).multiple_organizations;
+
+    return hasMultipleOrganizations || organizationsEnabled
```

Comment:
If the deployment has multiple organizations. but the organization's feature is not enabled, should this return true? The use case I'm seeing is a deployment that had access to enterprise and does not have it anymore after the trial.